.travis/apisix_cli_test.sh

```diff
@@ -52,3 +52,17 @@ if [ ! $? -eq 0 ]; then
 fi
 
 echo "passed: change default ssl port"
+
+# check nameserver imported
+i=`grep  -E '^nameserver[[:space:]]+(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4]0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[[:space:]]?$' /etc/resolv.conf | awk '{print $2}'`
+for ip in $i
+do
+  echo $ip
+  grep $ip conf/nginx.conf > /dev/null
+  if [ ! $? -eq 0 ]; then
+    echo "failed: system DNS "$ip "unimported"
+    exit 1
+  fi
+done
+
```

Comment:
These are not test cases, because no specific inputs and expected outputs, it ’s completely a black box

Reply:
This use case is used to test whether local DNS has been imported into nginx.conf

Reply:
@moonming  Are there other ways to test this feature? or remove the test case?